Makefile

```diff
@@ -1,106 +1,118 @@
-ifdef OS
-# WINDOWS version
-	LIBFLAGS = -lmingw32 -lSDL2main -lSDL2
+ifeq ($(OS),Windows_NT)
+	#INCLUDE_PATHS specifies the additional include paths we'll need
+	INCLUDE_PATHS = -IC:\mingw_dev_lib\include
+
+	#LIBRARY_PATHS specifies the additional library paths we'll need
+	LIBRARY_PATHS = -LC:\mingw_dev_lib\lib
+
+	#COMPILER_FLAGS specifies the additional compilation options we're using
+	# -w suppresses all warnings
+	# -Wl,-subsystem,windows gets rid of the console window
+	COMPILER_FLAGS = -w -Wl,-subsystem,windows
+
+	#LINKER_FLAGS specifies the libraries we're linking against
+	LIBFLAGS = -lmingw32 -lSDL2main -lSDL2 -lSDL2_ttf -lSDL2_image
 else
 	LIBFLAGS = -l sdl2 -l sdl2_ttf
 endif
 
-CC=gcc 
+#CC specifies which compiler we're using
+CC = gcc
+
+
 
-INCLUDES = include/map.h include/menu.h include/single_player.h include/map_textures.h include/multi_player.h
+INCLUDES = $(INCLUDE_PATHS) -Iinclude
```

Comment:
The following line should be there as these files are the prerequisites. 
INCLUDES = include/map.h include/menu.h include/single_player.h include/map_textures.h include/multi_player.h
If you remove that the .h files will not include at time of compilation.

The following line make it dependent to windows!
INCLUDES = $(INCLUDE_PATHS) -Iinclude